packages/browser-wallet/src/popup/shared/utils/transaction-history-types.ts

```diff
@@ -0,0 +1,52 @@
+import { AccountTransactionType } from '@concordium/web-sdk';
+
+/**
+ * The interface for the result of a query to get historical transactions for
+ * an account, e.g. from the wallet proxy.
+ */
+export interface TransactionHistoryResult {
+    /** full indicates that the number of transactions received is equal to the limit given, i.e. it indicates that there could be additional transactions available. */
+    full: boolean;
+    transactions: BrowserWalletTransaction[];
+}
+
+export enum TransactionStatus {
+    /** On-chain but with an error */
+    Failed = 'failed',
+    /** On-chain and successful */
+    Finalized = 'finalized',
+}
+
+export enum RewardType {
```

Comment:
Coincidentally I added the foundation account which caused the transactions not to load because it had 2 additional types, not that it matters. :upside_down_face: 
        
        paydayFoundationReward        
        platformDevelopmentCharge

Reply:
We will risk it.